AK/JsonParser.cpp

```diff
@@ -212,6 +212,8 @@ Optional<JsonValue> JsonParser::parse_number()
             whole = number.value();
         }
 
+        if (!fraction_string.to_uint().has_value())
+            return {};
         int fraction = fraction_string.to_uint().value();
```

Comment:
Good fix! But let's put the result of `to_uint()` in a local variable here so we don't do string-to-integer conversion twice :)

Reply:
Done.